.github/workflows/continuous-integration-e2e.yaml

```
@@ -47,6 +47,7 @@ jobs:
     - name: 🔬 Test - e2e - wallet
       run: |
         yarn workspace @cardano-sdk/e2e test:wallet
+        yarn workspace @cardano-sdk/e2e test:stake-pool
```

Do we need a separate test category for each provider? How about
```suggestion
        yarn workspace @cardano-sdk/e2e test:providers
```

---

I think... it would be hard to tell you're wrong 😃